setup.py

```diff
@@ -54,7 +54,7 @@ def run_tests(self):
         "cachetools >= 1.0.0",
         "Mopidy >= 1.1.2",
         "Pykka >= 1.1",
-        "pydora >= 1.13,<2",
+        "pydora >= 2,<3",
```

Comment:
Just to keep to the formatting elsewhere:

```suggestion
        "pydora >= 2, < 3",
```

Is there any specific reason to add an upper bound to the dependency? If not, I'd drop the upper bound.

Reply:
The reason for the upper bound stems from the recommendation in the pydora [README](https://github.com/mcrute/pydora#compatability):

> This package uses semantic versioning. The API is guaranteed to be stable within a major version release. Please constrain your dependencies to major versions.